estate/models/res_country.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+class CountryState(models.Model):
+    _name = 'res.country.state'
+    _description = 'Country State'
+    _order = 'code'
+
+    country_id = fields.many2one ('res.country', string='Country', required=True)
```

Comment:
```suggestion
    country_id = fields.many2one('res.country', string='Country', required=True)
```
Keep you code clean & structured.